### PR TITLE
Add overlay classes

### PR DESCRIPTION
**Pull request by storuky**
In my case I needed customize overlay for active modal. For example: Some modals should be opened in center, some modals as a right-side addon.
Could be useful.

[x] Add overlayClasses property

### CODE REVIEW

File: src/ModalsContainer.vue [hunk 1]
```diff
@@ -34,13 +35,16 @@ export default {
       if (!config.name) {
         config.name = '_dynamic-modal-' + id
       }
+
+      this.modals = this.modals.filter(modal => modal.config.name != config.name)
```
Comment (euvl) [inline src/ModalsContainer.vue]: Why is this needed?

File: src/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ const Plugin = {
     Vue.prototype.$modal = {
       show (modal, paramsOrProps, params, events = {}) {
         if (typeof modal === 'string') {
+          Plugin.event.$emit('toggle', modal, false, paramsOrProps)
```
Comment (euvl) [inline src/index.js]: And this

**Comment by storuky** [comment]
Yes, forgot about it. I think this is a right way if you close all modals with the same name before you open new one.
For example: you have many objects and `PropertiesEditorModal` to manage them. You open modal by clicking on object. If you already have a modal with the same(`PropertiesEditorModal`) it should be closed. Now it just spawn new one over the old.

I can remove this code part and make a pull-request just with overlayClasses.
